package.json

```diff
@@ -3,6 +3,12 @@
   "version": "1.0.0",
   "description": "ES7 decorator for adding $inject annotations for Angular 1.x",
   "main": "./src",
+  "files": [
+    "LICENSE",
+    "package.json",
+    "README.md",
```

Comment:
you can just do `src/` since the others are automatically included by npm no matter what